app/helpers/arclight_helper.rb

```diff
@@ -34,7 +34,7 @@ def collection_count
   end
 
   def repositories_active?
-    controller_name == 'repositories'
+    controller_name == 'repositories' && action_name == 'index'
```

Comment:
Ah. my inclination was that the "Repositories" link in the header would be active to indicate a subpath of "Repositories". This behavior is similar to the way that "Collections" works.